### ISSUE TITLE
New logo proposal for UnCiv application

### ISSUE THREAD

**Issue by ammarraisafti** [yairm210/UnCiv#160]
Hi admin, I want to offer a simple new logo design for UnCiv application. This logo is my special design for you. If you are willing to accept it, you simply reply to this proposal. I will send you a new logo of this file. And if you want other color variations , you can tell me. thank you

![unciv new logo](https://user-images.githubusercontent.com/37721970/41511713-65848010-72a6-11e8-9796-0f20d231768e.png)

**Comment by yairm210** [comment]
First of all, thank you for your time and effort!
To be honest, it looks like it's an icon for a different kind of game, Achikaps-style resource management, or a networking game. The kind of games where you have levels, and a game doesn't take hours.
Those are good games, but not at all how Civ works, and I'd rather have a logo that says something only to those who know the franchise, than a logo that gives a wrong impression of the game.

**Comment by ammarraisafti** [comment]
Thank you very much for your feedback. I really appreciate it. I will try
even harder to redesign the new logo with a good impression of this game,
if you allow it. What is your opinion?

Pada tanggal Sen, 18 Jun 2018 12:30, yairm210 <notifications@github.com>
menulis:

**Comment by yairm210** [comment]
Hmm
On the one hand, I don't want to tell you 'go for it' because there's a good chance that in the end I'll just stick with what I have :/
But on the other hand, there's a chance that your next logo will have me saying 'that's exactly it!' so I don't want to discourage you either
... Worth a shot, I guess?

If I was designing the logo, since it's a hex-based game, I would have the background be a hexagon, and on it the 6 resources (food, gold, production, science, culture, happiness)

... But as previously stated I'm not a designer so maybe I don't know what I'm talking about :-]

Monday, 18 June 2018, 09:11a.m. +03:00 from ammarraisafti notifications@github.com<mailto:notifications@github.com>:

Thank you very much for your feedback. I really appreciate it. I will try
even harder to redesign the new logo with a good impression of this game,
if you allow it. What is your opinion?

Pada tanggal Sen, 18 Jun 2018 12:30, yairm210 <notifications@github.com<mailto:notifications@github.com>>
menulis:

**Comment by ammarraisafti** [comment]
Hello sir.

I have attempted to create a new logo for the UnCiv app.
Here is a Logo I designed for UnCiv apps, hope you like it. If you have any
other suggestions, you can tell me, I would love to do it for you.

What is your opinion?

Best regards

Pada tanggal Sen, 18 Jun 2018 14:41, yairm210 <notifications@github.com>
menulis:

**Comment by ammarraisafti** [comment]
I have attempted to create a new logo for the UnCiv app.
Here is a Logo I designed for UnCiv apps, hope you like it. If you have any other suggestions, you can tell me, I would love to do it for you.

![new logo for app unciv po](https://user-images.githubusercontent.com/37721970/42003353-4578087c-7a95-11e8-8167-a60fda027745.png)

What is your opinion?

Best regards

**Comment by yairm210** [comment]
The logo is really cluttered, there are 8 (!) Different colors...
But the logo with only the 6 white hexagons inside looks pretty good, except for the intersection parts
A single dark hexagon with 6 white ones would look good, I think :)

Thursday, 28 June 2018, 01:37a.m. +03:00 from ammarraisafti notifications@github.com<mailto:notifications@github.com>:


I have attempted to create a new logo for the UnCiv app.
Here is a Logo I designed for UnCiv apps, hope you like it. If you have any other suggestions, you can tell me, I would love to do it for you.

[new logo for app unciv po]<https://user-images.githubusercontent.com/37721970/42003353-4578087c-7a95-11e8-8167-a60fda027745.png>

What is your opinion?

Best regards

**Comment by ammarraisafti** [comment]
Thank you very much for your feedback, I really appreciate it. I really enjoy this app, and have played this game for several months. Thanks for your wonderful work. It is amazing. This game is very interesting.

Here is an update that I did, like the advice you gave. Do they look good ?.
![100](https://user-images.githubusercontent.com/37721970/42029912-42702b8a-7afb-11e8-983f-272f2058b2a0.png)


This logo I design to give a new impression on your application. I hope you like it.

Are you going to use this logo on the next application release?

**Comment by ammarraisafti** [comment]
Thank you very much for your feedback, I really appreciate it. I really enjoy this app, and have played this game for several months. Thanks for your wonderful work. It is amazing. This game is very interesting.

Here is an update that I did, like the advice you gave. Do they look good ?.

This logo I design to give a new impression on your application. I hope you like it.

![new logo for app unciv cdrrrr](https://user-images.githubusercontent.com/37721970/42030627-a3a2db30-7afd-11e8-8b6a-5aa1626fddf5.png)

**Comment by ammarraisafti** [comment]
Thank you for your feedback. I'm trying to do it, and it's not over yet. I try to change it or make a simple design and give a good impression. So users can find it easily. And I am very glad for your feedback. When it's done, I'll let you know.

Oh yeah, what do you think with this one logo ?.
![next 1](https://user-images.githubusercontent.com/37721970/42042330-22fce5b8-7b1e-11e8-89c0-6c33301934a2.png)


I tried to design 6 hexagons on the side, and provided a bright color, with a background of two hexagons with a dark color. It looks like it will look modern and simple.

**Comment by yairm210** [comment]
Filius's logo makes me think that direction could work
But I also the white-on-dark (although it doesn't seem to be centred)
So maybe if you take the white+dark and put an equivalent flat icon on each hexagon -
Purple music note,
Yellow smily face,
Blue beaker,
Brown cog,
Green apple,
Golden coin

Thoughts?

**Comment by ammarraisafti** [comment]
A pretty good suggestion. I have fixed it. What is your opinion?

![next 1](https://user-images.githubusercontent.com/37721970/42230007-177bd364-7f12-11e8-8f72-8ef4cf4f2418.png)

**Comment by Filius-Patris** [comment]
That one's really great.
I love the logo really, it is nearly ready for the Play Store, I think. But first let's consider following things:
- The guy in the middle looks very fat. Or at least not as elegant as the one before.
- The lines between the outer tiles seem ok, but maybe you could make them about 2 pixel thicker. I'm really not sure. Thoughts?
- The gold seems too real compared too the rest of the icons. I really don't have a much better idea, but maybe just plain yellow bars without glitter?
- Last point: What would look better: Rounded edges (in the outer hexagon) or the peaky edges like your current proposal? Just try it out. You will see it yourself, if it is an improvement or not. But keep in mind, this icon will also appear on the phone's homescreen.

Don't get discouraged! I really love your suggestion. It is a great logo and I can imagine this on the Play Store and on my Phone. I just wanted to leave you these points to keep in mind.

**Comment by yairm210** [comment]
![unciv icon v1](https://user-images.githubusercontent.com/8366208/42241780-6d26de06-7f14-11e8-8893-76a2996386f6.png)

That's my attempt, I don't like that the smiley face is less noticable on the light background but I don't like the smiley when it's darker either.

**Comment by ammarraisafti** [comment]
Wow, the important point points you suggest, it's amazing. I really appreciate your input. I'm so glad you like it. What about these logos, Do they look good?

![next 1](https://user-images.githubusercontent.com/37721970/42243301-10f01a9a-7f3b-11e8-8c17-b5c9b3854538.png)

**Comment by ammarraisafti** [comment]
What do you think with this one logo. Does this seem to give a good impression?

Improvements will continue to be made, if you have any other suggestions.

![golom leuh](https://user-images.githubusercontent.com/37721970/42251012-00cc46d8-7f5f-11e8-8f60-10d40e84776b.png)

**Comment by Filius-Patris** [comment]
Feist of all: I just saw, th lines between the outer hexagons were ok. Either you make the inner lines thicker, or the outer lines between the tiles thinner. But in my opinion, this looks like an earthquake.

Then, I find the logos of yairm210 more clear. The only thing, I would correct (beside the smiley, which hurts the eye, you're right) is the note. It isn't centered in it's tile.

But for the man in the middle (sorry, I couldn't resist this expression ;) ) I like more the one from ammarraisafti, because the guy from yairm210's suggestion looks mor line an actual figure on a game like a board game. This guves the impression, that *you* have to move in this game.

One more thing about the lines between the outer tiles: In your 5th proposal, ammarraisafti, you drew them one pixel thick and a little darker than the tiles. I'd draw them about 3 pixel thick (remember: that logo will be downscaled for to be on the homescreen), but with the same color.

So, we're left with the smiley problem, and I have no solution to that too...

**Comment by yairm210** [comment]
![unciv icon v2](https://user-images.githubusercontent.com/8366208/42288527-b6ff0418-7fc3-11e8-96c9-45b478840904.png)

So!
I darkened the icons a little but to increase the contrast to the background
Centered the music note (thanks!)
The single unit icon is replaced with a "population" icon per your suggestion
And took a different smiley icon which is outline-based, so I could fill it with color

Thoughts?

**Comment by Filius-Patris** [comment]
I still see the old icon on the Google Play Store.

Using Android 4.4.2

**Comment by yairm210** [comment]
Way back when, I checked how the icon change affected the install rate, and it was dramatic (about 20%)

![image](https://user-images.githubusercontent.com/8366208/49497928-3b522700-f872-11e8-9278-1cf502daac6d.png)

The think is, the icon and the name are the first things you see about a game, before you see the about.

Having the reverse-Civ icon makes Civ people think "this looks like what I'm aiming for", which an unrelated symbol just doesn't do. It's a question of branding.

That was when we only got like ~20 installs per day, now we're at ~150, so we can try another test run now
